src/pymor/models/iosys.py

```diff
@@ -627,38 +627,109 @@ def to_abcde_files(self, files_basename):
         if E is not None:
             _mmwrite(Path(files_basename + '.E'), E)
 
-    def _compute(self, solution=False, output=False, solution_d_mu=False, output_d_mu=False,
-                 solution_error_estimate=False, output_error_estimate=False,
-                 output_d_mu_return_array=False, mu=None, **kwargs):
+    def compute(self, solution=False, output=False, solution_d_mu=False, output_d_mu=False,
+                solution_error_estimate=False, output_error_estimate=False, output_d_mu_return_array=False,
+                output_error_estimate_return_vector=False, mu=None, input=None, **kwargs):
```

Comment:
Sorry for being late to the party, but why overwrite `compute` instead of `_compute`? In principle, `Model.compute` is designed to be never overwritten directly (at least not Models shipped with pyMOR). If there is an issue with `_compute`, we should discussed whether this issue can be resolved ..

Reply:
I think the issue was that `Model.compute` does not allow only computing the output (without computing the solution).

Reply:
I see. Then I think the implementation of `compute` should be fixed. It seems like the check whether to call `_compute_solution` or not needs to be corrected.

Reply:
@sdrave maybe like [so](https://github.com/pymor/pymor/pull/2203/commits/4cb844658df97e2331da182995524fd262536802)?